### DISCUSSION TITLE
I Need a list of major bugs for reported on the Eclipse Theia

### DISCUSSION THREAD

**Discussion by Maheshkumar0018** [eclipse-theia/theia#12325] [Q&A]
I Need a list of major bugs for reported on the Eclipse Theia GitHub repository, along with their issue numbers, descriptions, and current status. bugs that related to Embedded

**Comment by colin-grant-work** [comment]
Thanks for your question. Since Theia does not have a suite of features specifically tailored to embedded use cases, there is no corresponding label applied to issues. The best proxy is likely to look for `debug` related issues, as debugging embedded code comes with special challenges. You might also be interested in any issues related to language features for C or other languages you use.

CC: @thegecko, since this may be more up your alley.

**Comment by thegecko** [comment]
You can also look at the extensions maintained which are more tailored for embedded development:

https://github.com/eclipse-cdt-cloud

However, I'd be keen to understand what you want to extract from this. Is the question more along the lines of

> Is the Theia platform ready to build an embedded-focus product?
